glm/gtc/packing.inl

```diff
@@ -9,6 +9,9 @@
 #include "../detail/type_half.hpp"
 #include <cstring>
 #include <limits>
+extern "C" {
+#include <endian.h>
```

Comment:
https://github.com/mikepb/endian.h/blob/0f885cbba627efe9b8f763e1c2872e904fe0c0b1/endian.h has equivalents to `endian.h` for abunch of platforms, if you're still interested in completing this.